aicsimageio/__init__.py

```diff
@@ -1,7 +1,4 @@
-from aicsimageio.version import MODULE_VERSION
-
 from .aics_image import AICSImage  # noqa: F401
+from .aics_image import imread  # noqa: F401
 
-
-def get_version():
-    return MODULE_VERSION
+__version__ = "3.0.0"
```

Comment:
Can you explain if / how this gets auto-incremented?  You can just tell me later if it doesn't make sense to write it here.

Reply:
Also, do we have code that depends on get_version() ?

Reply:
We have one instance of `get_version()` being used:

https://aicsbitbucket.corp.alleninstitute.org/plugins/servlet/search?q=aicsimageio.get_version()

This gets incremented when you run the following commands (also found in the `CONTRIBUTING.md` file):
> ## Deploying

> A reminder for the maintainers on how to deploy.
> Make sure all your changes are committed.
> Then run:

> `$ bumpversion patch` possible: major / minor / patch
> `$ git push`
> `$ git push --tags`

> Travis will then deploy to PyPI if tests pass.